modules/samples/oauth2/custom-grant/src/main/java/org/wso2/sample/identity/oauth2/grant/password/RoleBasedPasswordGrant.java

```diff
@@ -40,20 +40,22 @@
     @Override
     public boolean validateGrant(OAuthTokenReqMessageContext tokReqMsgCtx) throws IdentityOAuth2Exception {
 
+        boolean superAuthorized = super.validateGrant(tokReqMsgCtx);
```

Comment:
let's change the variable name to `grantValidated`

Reply:
fixed in https://github.com/wso2/product-is/pull/7357